Source/WebKit/UIProcess/API/Cocoa/_WKTextExtraction.swift

```diff
@@ -33,15 +33,76 @@ internal import WebKit_Internal
 // FIXME: Adopt `@objc @implementation` when support for macOS Sonoma is no longer needed.
 // FIXME: (rdar://110719676) Remove all `@objc deinit`s when support for macOS Sonoma is no longer needed.
 
+private func eventListenerTypesAsArray(eventListeners: WKTextExtractionEventListenerTypes) -> [String]? {
```

Comment:
```suggestion
private static func eventListenerTypesAsArray(_ eventListeners: WKTextExtractionEventListenerTypes) -> [String]? {
```

Reply:
This suggestion also breaks the build.

Reply:
```
…/OpenSource/Source/WebKit/UIProcess/API/Cocoa/_WKTextExtraction.swift:73:16: error: static methods may only be declared on a type
private static func eventListenerTypesAsArray(eventListeners: WKTextExtractionEventListenerTypes) -> [String] {
```

> And did you make the obj-c declaration of this method

I did not, because there is no ObjC declaration of this method.

Reply:
Ohhh sorry didn’t realize this was outside the type. My bad!